website/src/SUMMARY.md

```diff
@@ -44,37 +44,42 @@
-  - [new-branch-type](preferences/new-branch-type.md)
-  - [observed-branches](preferences/observed-branches.md)
-  - [observed-regex](preferences/observed-regex.md)
-  - [offline](preferences/offline.md)
-  - [parent](preferences/parent.md)
-  - [parked-branches](preferences/parked-branches.md)
-  - [perennial-branches](preferences/perennial-branches.md)
-  - [perennial-regex](preferences/perennial-regex.md)
-  - [prototype-branches](preferences/prototype-branches.md)
-  - [push-hook](preferences/push-hook.md)
-  - [push-new-branches](preferences/push-new-branches.md)
-  - [ship-delete-tracking-branch](preferences/ship-delete-tracking-branch.md)
-  - [ship-strategy](preferences/ship-strategy.md)
-  - [sync-feature-strategy](preferences/sync-feature-strategy.md)
-  - [sync-perennial-strategy](preferences/sync-perennial-strategy.md)
-  - [sync-prototype-strategy](preferences/sync-prototype-strategy.md)
-  - [sync-tags](preferences/sync-tags.md)
-  - [sync-upstream](preferences/sync-upstream.md)
+  - [Branches]()
+    - [Contribution branches](preferences/contribution-branches.md)
+    - [Contribution regex](preferences/contribution-regex.md)
+    - [Default branch type](preferences/default-branch-type.md)
+    - [Feature regex](preferences/feature-regex.md)
+    - [Main branch](preferences/main-branch.md)
+    - [Observed branches](preferences/observed-branches.md)
+    - [Observed regex](preferences/observed-regex.md)
+    - [Parked branches](preferences/parked-branches.md)
+    - [Perennial branches](preferences/perennial-branches.md)
+    - [Perennial regex](preferences/perennial-regex.md)
+    - [Prototype branches](preferences/prototype-branches.md)
+  - [Create]()
+    - [Create prototype branches](preferences/create-prototype-branches.md)
+    - [New branch type](preferences/new-branch-type.md)
+    - [Push new branches](preferences/push-new-branches.md)
+  - [Hosting]()
+    - [Development remote](preferences/dev-remote.md)
+    - [Hosting platform](preferences/hosting-platform.md)
+    - [Origin hostname](preferences/hosting-origin-hostname.md)
+    - [Bitbucket access token](preferences/bitbucket-app-password.md)
+    - [Bitbucket username](preferences/bitbucket-username.md)
+    - [Gitea token](preferences/gitea-token.md)
+    - [GitHub token](preferences/github-token.md)
+    - [GitLab token](preferences/gitlab-token.md)
+  - [Ship]()
+    - [Delete tracking branch](preferences/ship-delete-tracking-branch.md)
+    - [Ship strategy](preferences/ship-strategy.md)
+  - [Sync]()
+    - [Feature sync strategy](preferences/sync-feature-strategy.md)
+    - [Perennial sync strategy](preferences/sync-perennial-strategy.md)
+    - [Prototype sync strategy](preferences/sync-prototype-strategy.md)
+    - [Run pre-push hook](preferences/push-hook.md)
+    - [Sync tags](preferences/sync-tags.md)
+    - [Sync with upstream](preferences/sync-upstream.md)
+  - [Offline mode](preferences/offline.md)
+  - [Branch lineage](preferences/parent.md)
```

Comment:
If you don't rename "Branches" to "Branch Types", could this go under "Branches"?

Reply:
I don't feel strongly about this one, but I think it is better to leave this at the bottom. The named sections like "Branches" refer to preferences that the user can change, and the "Branch lineage" page exists just to say "you can ignore these Git config entries, they're Git Town internal data".